docs/DeveloperGuide.md

```diff
@@ -404,6 +404,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Appendix: Planned Enhancement**
```

Comment:
Might need to specify team size for PE purpose.

<img width="492" alt="image" src="https://github.com/AY2324S2-CS2103T-T11-2/tp/assets/73763719/f65b90ae-9018-45ca-8d84-b19a17517cee">

Example:
<img width="314" alt="image" src="https://github.com/AY2324S2-CS2103T-T11-2/tp/assets/73763719/694d9b8e-973d-4f12-bcb2-c5b53c43a801">

Reply:
Also, minor grammar nitpick but, I think it should be plural for the `Enhancement` in the section header.

Reply:
Some past year teams don't do that https://ay2324s1-cs2103t-t11-2.github.io/tp/DeveloperGuide.html#appendix-planned-enhancement . I don't think testers have any need to know that either (it's just for grading purpose)

Reply:
That's fair, but I think is safer to add in since the instructions say to clearly state it for this sem.